AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -130,7 +132,12 @@ internal final class ComponentManager {
         return component
     }
     
-    private func createMBWayComponent(_ paymentMethod: MBWayPaymentMethod) -> MBWayComponent {
+    private func createMBWayComponent(_ paymentMethod: MBWayPaymentMethod) -> MBWayComponent? {
+        guard configuration.clientKey != nil else {
+            // swiftlint:disable:next line_length
+            adyenPrint("Failed to instantiate MBWayComponent because client key is not configured, Please supply the client key in the PaymentMethodsConfiguration.")
```

Comment:
Should we mention DrpoInACtionComponent here, too?

Reply:
here its means that they are using dropInComponent, and `DropInComponent` is instantiating the `DrpoInACtionComponent` and making sure the client key is supplied to it.